Blizzard.IO.RabbitMQ/Implementaions/NetqRabbitConsumer.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using Blizzard.IO.Core;
+using Blizzard.IO.RabbitMQ.Entities;
+using EasyNetQ;
+using EasyNetQ.Topology;
+
+namespace Blizzard.IO.RabbitMQ.Implementaions
```

Comment:
I dont think we want this namespace name. It is not in the conventions.
The whole point of Blizzard.IO.Core is to contain the abstractions and in Blizzard.IO.RabbitMQ to contain the implementations, I think it is redundant and adding to code length with no benefits of readabillity

Reply:
f